packages/core/src/log/shared.ts

```diff
@@ -300,6 +300,8 @@ export class ZWaveLogContainer extends winston.Container {
 		const ret = new DailyRotateFile({
 			filename: this.logConfig.filename,
 			datePattern: "YYYY-MM-DD",
+			createSymlink: true,
+			symlinkName: this.logConfig.filename.replace(`_%DATE%`, "_current"),
```

Comment:
What happens if `_%DATE%` is missing from the filename, as if someone customized the filename via the driver options?

Reply:
N/M, I think it's appended to the filename elsewhere.

Reply:
yup